index.html

```diff
@@ -4,13 +4,36 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Instagram Clone</title>
-    <!-- don't forget to link your styles -->
-  </head>
+    <link rel="stylesheet" href="/styles/style.css"> 
+ </head>
   <body>
-    Instagram Capture and Share the World's Moments Instagram is a fast, beautiful and fun way to share your life with
-    friends and family. Take a picture or video, choose a filter to transform its look and feel, then post to Instagram
-    &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
-    Oh yeah, did we mention it's free? Download on the App Store Get it on Google Play About Us Support Blog Press API
-    Jobs Privacy Terms &copy; 2014 Instagram
+    <section class="blue"></section>
+    <section class="phones">
+      <img src="/images/phones.png" alt="">
+    </section>
+    <section class="logo">
+      <div class="logo-button">
+      <img class="brand" src="images/brand.png" alt="">
+      <div class="log-in">
+      <img class="home" src="images/home.png" alt="">
+      <p>Log In</p>
```

Comment:
El problema que tuviste con el login es que colocaste un parrafo, cosa que no está mal, pero recuerda que los parrafos son elementos de bloque cuya naturaleza es buscar ocupar todo el ancho disponible y saltar de linea. Por lo cual la solucion era colocar el texto suelto:

```suggestion
     Log In
```